src/components/backend-ai-app-launcher.ts

```diff
@@ -499,13 +499,19 @@ export default class BackendAiAppLauncher extends BackendAIPage {
         }
       }
     });
-    if (globalThis.isElectron && !appServices.includes('vscode')) {
-      this.appSupportList.push({ // Force push vscode
-        'name': 'vscode',
-        'title': 'VS Code',
-        'category': '2.Development',
-        'redirect': '',
-        'src': './resources/icons/vscode.svg'
+
```

Comment:
Somewhat I cannot see two version of vscode (vscode execution in local, vscode with web application.) in desktop app version under the condition; the latest commit of this PR branch.
I tested with [`23.03.0`](https://github.com/lablup/backend.ai/releases/tag/23.03.0) version of Backend.AI Core.

Reply:
<img width="369" alt="Screenshot 2023-04-10 at 3 25 28 PM" src="https://user-images.githubusercontent.com/46954439/230840644-50e3f5bb-682f-4f5f-be85-f72aaf3a58ef.png">